fmriprep/workflows/base.py

```diff
@@ -11,21 +11,18 @@
 
 """
 
-import sys
 import os
+import sys
 from copy import deepcopy
 
-from niworkflows.nipype.pipeline import engine as pe
 from niworkflows.nipype.interfaces import utility as niu
+from niworkflows.nipype.pipeline import engine as pe
 
-from ..interfaces import (
-    BIDSDataGrabber, BIDSFreeSurferDir, BIDSInfo, SubjectSummary, AboutSummary,
-    DerivativesDataSink
-)
+from ..info import __version__
+from ..interfaces import (AboutSummary, BIDSDataGrabber, BIDSFreeSurferDir,
+                          BIDSInfo, DerivativesDataSink, SubjectSummary)
 from ..utils.bids import collect_data
 from ..utils.misc import fix_multi_T1w_source_name
-from ..info import __version__
-
```

Comment:
I'm not sure if these are changes you've made or if they were changed in parallel and you're accidentally reverting them, but could you check these lines out from `master`, so that only new or removed imports show up in the diff?